src/libs/actions/Policy.ts

```diff
@@ -249,6 +257,9 @@ function updateLastAccessedWorkspace(policyID: OnyxEntry<string>) {
     Onyx.set(ONYXKEYS.LAST_ACCESSED_WORKSPACE_POLICY_ID, policyID);
 }
 
+function hasValidCurrencyForDirectReimbursement(currency: string) {
```

Comment:
Maybe a bit cleaner?
```suggestion
function hasCurrencySupportedForDirectReimbursement(currency: string) {
```